### PR TITLE
added tests for filters on search page

### PR DESCRIPTION
**Pull request by pbadak**

### CODE REVIEW

File: suites/test_search_beta.py [hunk 1]
```diff
@@ -99,3 +99,38 @@ def modules_assemblies_list(self):
...
+        utilities.click_element(self.driver, By.CLASS_NAME, locators.STATUS_FILTER_CLASS_NAME)
+        utilities.click_element(self.driver, By.CLASS_NAME, locators.DRAFT_STATUS_CLASS_NAME)
+        check_that("Draft Status filter chip is displayed on search page", utilities.find_element(self.driver, By.CLASS_NAME,
+                          locators.STATUS_TOOLBAR_CHIP_CLASS_NAME).is_displayed(), is_true())
+        last_published_date_list = utilities.find_elements_by_XPATH(self.driver, locators.LAST_PUBLISHED_DATE_XPATH)
+        published_date_list = []
+        for i in last_published_date_list:
+            published_date_list.append(i.text)
+        for i in published_date_list:
+            check_that("Draft status for all titles", i, contains_string("-"))
```
Comment (noopurmath3) [inline suites/test_search_beta.py:114]: Should this be something like "Published date for all titles contains '-' "?

File: suites/test_search_beta.py [hunk 2]
```diff
@@ -99,3 +99,38 @@ def modules_assemblies_list(self):
...
+        utilities.click_element(self.driver, By.CSS_SELECTOR, locators.CLEAR_ALL_FILTER_CSS)
+        utilities.click_element(self.driver, By.CLASS_NAME, locators.CONTENT_TYPE_CLASS_NAME)
+        utilities.click_element(self.driver, By.CLASS_NAME, locators.CONCEPT_CONTENT_TYPE_FILTER_CLASS_NAME)
+        check_that("Content Type filter chip is displayed on search page",
+                   utilities.find_element(self.driver, By.CLASS_NAME,
+                                          locators.STATUS_TOOLBAR_CHIP_CLASS_NAME).is_displayed(), is_true())
+        utilities.wait(1)
+        check_that("concept content type filter is selected", utilities.get_text(self.driver, By.CSS_SELECTOR,
```
Comment (noopurmath3) [inline suites/test_search_beta.py:122]: Should be "'Concept' content type filter is selected"
similar changes for all the 3 tests

**Review by noopurmath3 — COMMENTED** [review]
Added a few minor comments
Can we also add tests for filtering published titles as well?
Other than that the changes look good to me

File: suites/test_search_beta.py (hunk 1 above)
Comment (pbadak) [reply suites/test_search_beta.py]: review changes acknowledged :) thanks for review @noopurmath3

File: suites/test_search_beta.py (hunk 2 above)
Comment (pbadak) [reply suites/test_search_beta.py]: done

**Pull request merged**